ansible/roles/base/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+
+---
+
+    - name: Update apt cache
+      apt: update-cache=yes cache_valid_time=86400
+      become: yes
+      become_method: sudo
```

Comment:
Thinking about it, I think what should happen is the become_method is set at the playbook level. Whether to escalate or not should still be attached to the task.